modules/serverless-configuring-startup-probing-serving.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative-serving/config-applications/startup-probes-for-serving.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-configuring-startup-probing-serving_{context}"]
+= Configuring startup probing
+
+For {ServerlessProductName} Serving, startup probes are not defined by default. You can define startup probes for your containers in your deployment configuration. 
+
+.Procedure
+
+* Define startup probes for your service by modifying your deployment configuration. The following example shows a configuration with two containers:
++
+.Example of defined starup probes 
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+...
```

Comment:
YAML truncated lines should be commented out, per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks

```suggestion
# ...
```

Same for other lines below